Tasks/AzureFileCopy/task.json

```diff
@@ -1,19 +1,20 @@
 ﻿{
-  "id": "94875CAE-C3AE-4540-88A5-2FA2440D8AFE",
+  "id": "EB72CB01-A7E5-427B-A8A1-1B31CCAC8A43",
```

Comment:
In S83 we had CopyFilesToBlob task. In S84 we have enhanced this Task to handle both blob and Azure VM copy and renamed as AzureFileCopy. Earlier check-in added this new capability as a separate Task with new GUID which was a mistake. With this check-in we are ensuring that CopyFilesToBlob Task is upgraded to AzureFileCopy. So fixing here to retain the same GUID.